teuthology/run.py

```diff
@@ -254,7 +254,7 @@ def get_initial_tasks(lock, config, machine_type):
         # upstream changed this to "clock" in f5d9712 but in openstack
         # we have the ability to sync clocks in the user-data script,
         # so we are keeping "clock.check" for now
-        init_tasks.append({'clock.check': None})
+        init_tasks.append({'clock': None})
```

Comment:
I guess the comment above can be dropped.

Reply:
done